src/client/DeploymentTable.tsx

```diff
@@ -330,7 +385,11 @@ export const renderMergedBy = (
   tableColumn: ITableColumn<IDeploymentField>,
   deployment: IDeploymentField
 ): JSX.Element => {
-  if (deployment.pr && deployment.mergedByName) {
+  if (
+    deployment.pr &&
+    deployment.mergedByName &&
```

Comment:
I'd recommend keeping as `if (deployment.pr && deployment.mergedByName)` as its more concise and does the same thing (empty string is falsy)